library/psa_crypto_se.c

```diff
@@ -83,4 +123,10 @@ void psa_unregister_all_se_drivers( void )
     memset( driver_table, 0, sizeof( driver_table ) );
 }
 
+
+
+/****************************************************************/
+/* The end */
+/****************************************************************/
```

Comment:
Please add ASCII art of a man taking a bow. It should fit very well with this "The end" comment.

Reply:
ASCII art request denied.

Reply:
:-(